src/main/haskell/kore/src/Kore/Step/Strategy.hs

```diff
@@ -198,28 +203,79 @@ seq a continue === a
 continue :: Strategy prim
 continue = Continue
 
-{- | Run a simple state machine.
+type ExecutionGraph config = (Graph.Node, Gr config (), [[ConfigNode config]])
 
-The transition rule may allow branching. Returns a tree of all machine states.
+data ConfigNode config = ConfigNode
```

Comment:
Please give this Haddocks; it seems like `ConfigNode` ties everything together, so it would be much easier to understand if this was well-documented.